CMakeLists.txt

```diff
@@ -571,6 +576,15 @@ if(NOT AVIF_CODEC_AOM_ENABLED
     message(WARNING "libavif: No decoding library is enabled.")
 endif()
 
+if(AVIF_BUILD_APPS
+   OR AVIF_CODEC_LIBGAV1_ENABLED
+   OR AVIF_ENABLE_COMPLIANCE_WARDEN
+   OR (AVIF_BUILD_TESTS AND (AVIF_ENABLE_FUZZTEST OR AVIF_ENABLE_GTEST))
```

Comment:
Optional: If the documentation for `AVIF_ENABLE_FUZZTEST` at lines 145-149 is accurate:

```
option(
    AVIF_ENABLE_FUZZTEST
    "Build avif fuzztest targets. Requires Google FuzzTest. Has no effect unless AVIF_BUILD_TESTS and AVIF_ENABLE_GTEST are ON."
    OFF
)
```

then the condition `AVIF_ENABLE_FUZZTEST OR AVIF_ENABLE_GTEST` can simplified to `AVIF_ENABLE_GTEST`.

Since this is an independent issue, you don't need to look into it in this pull request.

Reply:
I looked into this. I believe the documentation for `AVIF_ENABLE_FUZZTEST` is incorrect. It should say "Has no effect unless AVIF_BUILD_TESTS is ON." You can ignore this comment.